File: app/src/main/java/com/fpoly/project1/firebase/SessionUser.java
package com.fpoly.project1.firebase;

import android.net.Uri;

import com.fpoly.project1.firebase.controller.ControllerCustomer;
import com.google.android.gms.tasks.Tasks;

public class SessionUser {
    public static String sessionId;
    private static final ControllerCustomer controllerCustomer = new ControllerCustomer();

    public static void setId(String id) {
        sessionId = id;
    }

    public static Uri getAvatar() {
        try {
            return Tasks.await(
                    Firebase.storage
                            .child("/avatars/" + controllerCustomer.getSync(sessionId))
                            .getDownloadUrl()
            );
        } catch (Exception e) {
            return null;
        }
    }
}

File: app/src/main/java/com/fpoly/project1/firebase/SessionUser.kt
package com.fpoly.project1.firebase

import android.net.Uri
import com.fpoly.project1.firebase.model.Product
import com.google.android.gms.tasks.Task
import com.google.android.gms.tasks.Tasks

object SessionUser {
    var sessionId: String? = null
    val cart: MutableList<Pair<Product, Int>> = ArrayList()

    fun setId(id: String?) {
        sessionId = id
    }

    val avatar: Task<Uri>
        get() =
            Firebase.storage
                .child("/avatars/$sessionId.jpg")
                .downloadUrl
}
